docs/app/Examples/elements/Loader/Variations/LoaderInlineCenteredExample.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react'
+import { Loader } from 'stardust'
+
+export default () => <Loader active inline='centered' />
```

Comment:
Just for consistency, perhaps we should assign these to a const and export that.